src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -9,14 +9,35 @@ import DarCloseout from 'src/pages/progress_reports/DarCloseout';
 import SubmitProgressReport from 'src/pages/progress_reports/SubmitProgressReport';
 
 type ProgressReportApplicationProps = {
-    dar?: DataAccessRequest, // Dar will be empty if this is an application
-    parentDar?: DataAccessRequest, // Dar will be empty if this is view only
+    dar: DataAccessRequest, // corresponds either to the parent DAR for a new application or an existing readonly progress report
     datasets: Dataset[],
     readOnlyMode?: boolean
 };
 
-export const ProgressReportApplication = ({ dar, parentDar, datasets, readOnlyMode = true }: ProgressReportApplicationProps) => {
-    const [formState, setFormState] = useState<FormState>({});
+export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true }: ProgressReportApplicationProps) => {
+    const initialState = {
+        ...dar,
+        // additional state for summary section
+        ...(dar?.intellectualPropertySummary && {
```

Comment:
I'm not sure of a more compact way to approach this. This says: if the value is not undefined (which is the case in the readonly view), then set the correct key with the true / false value so these can be selected in the form.

Reply:
I think this is slightly more readable.. but, I'm not sure if it works the same to explicitly set it to undefined. 
```
const initialState = {
    ...dar,
    intellectualPropertyYesNo: dar?.intellectualPropertySummary ? !!dar.intellectualPropertySummary : undefined,
    publicationsYesNo: dar?.publications ? (dar.publications.length > 0) : undefined,
    presentationsYesNo: dar?.presentations ? (dar.presentations.length > 0) : undefined,
    dmiYesNo: dar?.dataManagementIncident?.incidents ? (dar.dataManagementIncident.incidents.length > 0) : undefined,
    closeoutYesNo: dar?.closeOutSupplement ? !!dar.closeOutSupplement : undefined,
};
```

Reply:
I'm not sure either. I try to avoid setting stuff to undefined, since we've seen state issues with that before.

Reply:
I think I'll keep it as is and if it's too confusing we can discuss another solution.